tests/test_updater.py

```diff
@@ -615,13 +615,13 @@ def test_idle(self, updater, caplog):
         # (bots) running simultaneously while testing in github actions.
         records = caplog.records.copy()  # To avoid iterating and removing at same time
         for idx, log in enumerate(records):
-            print(log)
+            print(idx, log)
             msg = log.getMessage()
             if msg.startswith('Error while getting Updates: Conflict'):
-                caplog.records.pop(idx)  # For stability
+                caplog.records.remove(log)  # For stability
 
-            if msg.startswith('No error handlers are registered'):
-                caplog.records.pop(idx)
+            elif msg.startswith('No error handlers are registered'):
+                caplog.records.remove(log)
 
```

Comment:
is this actually fixed or a next iteration of trying to fix it? If it's the first, I'd suggest to again move to a standalone PR

Reply:
hopefully completely fixed now